<ui version="4.0" >
 <class>udg::QDicomDumpMRWidgetBase</class>
 <widget class="QWidget" name="udg::QDicomDumpMRWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>630</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="m_labelProtocolName" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Protocol Name :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="m_labelProtocolNameValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="m_labelFieldOfView" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Field Of View :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="m_labelFieldOfViewValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="m_labelSliceThickness" >
        <property name="windowModality" >
         <enum>Qt::NonModal</enum>
        </property>
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Slice thickness :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="m_labelSliceThicknessValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="m_labelRatioFieldOfView" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Ratio Field Of View :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="m_labelRatioFieldOfViewValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="m_labelRepetitionTime" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Repetition Time :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="m_labelRepetitionTimeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="m_labelEchoTime" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Echo Time :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="m_labelEchoTimeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="m_labelInversionTime" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Inversion Time</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="m_labelInversionTimeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="m_labelFlipAngle" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Flip Angle :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QLabel" name="m_labelFlipAngleValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="m_labelReceiveCoil" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Receive Coil:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLabel" name="m_labelReceiveCoilValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" >
       <widget class="QLabel" name="m_labelNumberOfAverages" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Number Of Averages :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" >
       <widget class="QLabel" name="m_labelNumberOfAveragesValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame_2" >
     <property name="minimumSize" >
      <size>
       <width>630</width>
       <height>0</height>
      </size>
     </property>
     <property name="accessibleName" >
      <string/>
     </property>
     <property name="accessibleDescription" >
      <string/>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="m_labelPhilipsScanningTechnique" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="text" >
         <string>Scanning Technique :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="m_labelPhilipsScanningTechniqueValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="m_labelPhilipsImagePosition" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Image Position : ????</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="m_labelPhilipsImagePositionValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="m_labelPhilipsTurboFactor" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Turbo Factor :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="m_labelPhilipsTurboFactorValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="m_labelPhilipsEPIFactor" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>EPI Factor :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="m_labelPhilipsEPIFactorValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="m_labelPhilipsNumberOfStacks" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Number Of Stacks :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="m_labelPhilipsNumberOfStacksValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="m_labelPhilipsBFactor" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>B Factor :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="m_labelPhilipsBFactorValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="m_labelPhilipsDynamicScans" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Dynamic Scans :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="m_labelPhilipsDynamicScansValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
